content/events/2020/07/2020-07-16-women-in-federal-it-cyber.md

```diff
@@ -0,0 +1,95 @@
+---
+# View this page at https://digital.gov/event/2020/07/women-in-federal-it-cyber
+# Learn how to edit our pages at https://workflow.digital.gov
+slug: women-in-federal-it-cyber
+title: "Women In Federal IT & Cyber Conference Panel Discussion #2: Expanding Our Network"
+deck: ""
+kicker: "CIO Council"
+summary: "Panelists will discuss the importance of expanding our professional network outside of our silos to become stronger leaders."
+host: "CIO Council"
+event_organizer: "Digital.gov"
+registration_url: https://www.eventbrite.com/e/109198006176
+captions: https://www.captionedtext.com/client/event.aspx?EventID=4475926&CustomerID=321
+
+# start date
+date: 2020-07-16 14:00:00 -0500
+
+# end date
+end_date: 2020-07-16 15:00:00 -0500
+
+# see all topics at https://digital.gov/topics
+topics: 
+  - professional-development
+
+# Event platform (zoom, youtube_live, adobe_connect, google)
+event_platform: adobe_connect
+
+# YouTube ID
+youtube_id: 
+
+# Page weight: controls how this page appears across the site
+# 0 -- hidden
+# 1 -- visible
+weight: 0
+
+# Make it better ♥
+
+---
+
+## Presenters:
```

Comment:
I think the Presenters title should go a bit further down after the intro @ayerkes

Reply:
Okay I will update!